charts/ia-case-notifications-api/values.aat.template.yaml

```diff
@@ -1,4 +1,7 @@
 java:
   # Don't modify below here
   image: ${IMAGE_NAME}
-  ingressHost: ${SERVICE_FQDN}
\ No newline at end of file
+  ingressHost: ${SERVICE_FQDN}
+  environment:
+    IA_AIP_FRONTEND_URL: "https://ia-aip-frontend-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/"
```

Comment:
Can't remember exactly - is this used in the app to put into notifications sent out so that users can link to their case?  Is it used for other stuff?
I ask because with the azure dns changes we will end up with a url like `http://immigration-appeal.aat.platform.hmcts.net` rather than the .internal ones even in `aat`
The same may apply to XUI.

Reply:
Thanks @raghera 
Having had a look it's only used on the notifications that we send out.
But if that's the case then maybe I should create a ticket to update these again once those changes are in place. 
Then it should become `http://immigration-appeal.{{ .Values.global.environment }}.platform.hmcts.net` same for ExUI

Reply:
Front door in `aat` is already there and works.  Not sure about XUI though can keep that the same for now.